app/models/spree/payment/processing.rb

```diff
@@ -190,6 +186,10 @@ def authorization_action_required?
         cvv_response_message.present?
       end
 
+      def preauthorized?
+        response_code.presence&.match("pi_")
+      end
```

Comment:
this should be private right? sorry to be picky :sweat_smile:

Reply:
😁 👍